flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
I know, but it seems to be a limitation of Flutter at the moment. See the discussion at https://github.com/flutter/flutter/issues/41737